products/windows-nano-server.md

```diff
@@ -0,0 +1,121 @@
+
+---
+
+> Nano Server is a minimal installation option of Windows Server offering a smaller disc footprint with a smaller attack surface.
+> It was initially intended as even smaller installation option compared to Windows Server Core.
+> Starting with 1709 it was removed as installation option and going forward is only offered as Windows container base image.
+
+## [Servicing channels](https://learn.microsoft.com/windows-server/get-started/servicing-channels-comparison)
+There have been historically three primary release channels available:
+* Long-Term Servicing Channel (LTSC)
+* Semi-Annual Channel (SAC)
+* Annual Channel (AC)
+
+The Windows Server Semi-Annual Channel (SAC) [was retired on August 9, 2022](https://learn.microsoft.com/en-us/lifecycle/announcements/windows-server-20h2-retiring). There will be no future SAC releases of Windows Server.
+
+The Windows Server Annual Channel (AC) was released after the retirement of the Semi-Annual Channel (SAC). The first release of the Annual Channel (AC) was on October 24, 2023.
+
+### Long-Term Servicing Channel (LTSC)
+With the Long-Term Servicing Channel, a new major version of Windows Server is released every 2-3 years. Users are entitled to 5 years of mainstream support and 5 years of extended support. The Long-Term Servicing Channel will continue to receive security and non-security updates, but it will not receive the new features and functionality.
+
+### Semi-Annual Channel (SAC)
+Each release in this channel is supported for 18 months from the initial release. Most of the features introduced in the Semi-Annual Channel have been rolled up into the next Long-Term Servicing Channel release of Windows Server.
+In this model, Windows Server releases were identified by the year and month of release: for example, in 2017, a release in the 9th month (September) would be identified as version 1709. Fresh releases of Windows Server in the Semi-Annual Channel occurred twice each year. The support lifecycle for each release is 18 months. Starting with fall 2020 (20H2) releases, the identification changed. Instead of a month, the release is named based on the release cycle. For example: version 20H2, for a release in the second half of the year 2020.
+
+## Annual Channel (AC)
+Similar to Semi-Annual Channel (SAC), but releases occurred every 12 months.
+The support lifecycle for each release is 18 months of mainstream support, plus 6 months of extended support.
+
+## Extended Security Updates (ESU)
+
+For some releases, customers also have the option to purchase [Extended Security Updates (ESU)](https://learn.microsoft.com/lifecycle/faq/extended-security-updates).
+It includes Critical and/or Important security updates for a maximum of three years after the
+product's End of Extended Support date. Customers can also get technical support for the ESU period
+as long as they also have an active support plan.
```

Comment:
I would like to redact or remove this section because it is the same as in Windows Server. If all applies to Nano Server, better to reference the section in Windows Server instead of duplicate the details.

Reply:
I was not sure how this is usually dealt with here, if cross-linking is OK or if everything should be standalone. I will update this tomorrow morning.